appeals/web/src/server/appeals/appeal-details/change-appeal-type/change-appeal-type.mapper.js

```diff
@@ -367,13 +375,13 @@ export function resubmitConfirmationPage(appealData) {
 			{
 				type: 'html',
 				parameters: {
-					html: `<p class="govuk-body">The appellant has been emailed to ask them to resubmit their appeal with the correct type.</p>`
+					html: '<p class="govuk-body">The appellant has been asked to resubmit using the correct appeal type.</p>'
 				}
 			},
 			{
 				type: 'html',
 				parameters: {
-					html: `<p class="govuk-body">You can go <a href="/appeals-service/personal-list" class="govuk-link">back to your list</a> or <a href="/appeals-service/appeal-details/${appealData.appealId}" class="govuk-link">view the closed case</a>.</p>`
+					html: `<p class="govuk-body"><a class="govuk-link" href="/appeals-service/appeal-details/${appealData.appealId}" class="govuk-link">Go back to case details</a></p>`
```

Comment:
Do you need the <p> wrapper here? Don't think it was there in your previous change.